src/countdown/Countdown.tsx

```diff
@@ -0,0 +1,72 @@
+import "./_countdown.scss";
+
+import React, {useLayoutEffect} from "react";
+import classNames from "classnames";
+
+import useCountDownTimer from "../core/utils/hooks/useCountdownTimer";
+import List from "../list/List";
+import ListItem from "../list/item/ListItem";
+
+export interface CountdownProps {
+  testid: string;
+  startDate: Date;
+  countDownIntervalInSeconds?: number;
+  onEnd?: () => void;
+  customClassName?: string;
+}
+
+function Countdown({
+  testid,
+  startDate,
+  onEnd,
+  countDownIntervalInSeconds = 1,
+  customClassName
+}: CountdownProps) {
+  const countDownData = useCountDownTimer(startDate, countDownIntervalInSeconds);
+  let items: {id: "days" | "hours" | "minutes" | "seconds"; count: string | number}[];
```

Comment:
You can define a new type `CountDownItem` in this file, remove else from the conditions and give it to `items` as the first value.

```suggestion
  let items: CountDownItem[] = [
      {id: "minutes", count: countDownData.minutes},
      {id: "seconds", count: countDownData.seconds}
    ];
```

```
interface CountDownItem {
    id: "days" | "hours" | "minutes" | "seconds";
    count: string | number;
}
```